docs/infrastructure/private-clusters.mdx

```diff
@@ -0,0 +1,170 @@
+---
+id: private-clusters
+title:  Private clusters
+sidebar_label: Private clusters
+---
+
+Sometimes you could need to keep your kubernetes cluster private and not publicly reachable for security reasons.
+
+The reasons to do this are different:
+* you want to improve the cluster security;
+* company's policies impose this limitation on the infrastructure;
+* you don't expose APIs to be publicly used, but just for internal purpose;
+* ... and so on
+
+The ways to do this are different too, but in this document we will assume that you already own a private cluster, regardless of the used approach.
+
+What does it mean to have a private cluster for _Software Life Cycle Management_?
+
+On the Mia-Platform Console we "directly" interact with the kubernetes clusters in few pages:
+* __Runtime page__: represents the overview of the runtime environment status and allows the user to see and interact with elements on kubernetes (depends on the user permissions);
+* __Deploy page__: when enabling the smart-deploy toggle, the Mia-Platform Console query the kubernetes cluster to show the user differences between the actual release and the new release;
+* __Deploy page__: deploying a project will usually trigger the deploy pipeline to update the kubernetes resources with the new version.
+
+How can we handle the operations above if the cluster is private?
+
+# TL;DR;
+Following a recap table that shows you the different approaches and which use cases are they appropriate for.
+
+# Proxy is the way
+
+The proxy solution solved all the use cases listed above because using, for example, a bastion proxy, you can:
+* allow the _Runtime page_ to reach the cluster, authenticating it through on the proxy with an AuthN/AuthZ flow;
+* allow the _Deploy service_ to reach the cluster, authenticating it through on the proxy with an AuthN/AuthZ flow, to get the actual kubernetes resources and to publish new resources on the cluster.
+
+You can set the proxy you need to reach the cluster directly on the Mia-Platform Console, during the cluster creation in your company page.
+To do that you can:
+* Navigate, through the Mia-Platform Console, to your company settings page;
+* Click on the _Clusters_ menu;
+* Click on the _Add cluster connection_ button (or editing an existing one);
+* Insert cluster's details, until you reach the _Advanced_ section of the "Cluster creation stepper";
+* Insert the proxy's URL and, if you need it, a TLS certificate, as reported in [this documentation page](/development_suite/clusters-management/connect-and-manage-cluster.mdx#step-4-advanced).
+
+![Add cluster connection 4](img/add-cluster-connection-4.png)
+
+:::warning
+If you have a __Pure License__ Mia-Platform Console instance and the Mia-Platform Console is installed on the same runtime environment, you couldn't need the proxy because the communications between the Mia-Platform Console and the cluster will be internal.
```

Comment:
```suggestion
If you have a __Pure License__ Mia-Platform Console instance and the Mia-Platform Console is installed on the same runtime environment, you won't need the proxy because the communications between the Mia-Platform Console and the cluster will be internal.
```